_wiki/FAQ/Linux.md

```diff
@@ -101,19 +101,15 @@ Failed to initialize virtual tablet. (error code EACCES)
 
 Try disabling your filters one-by-one and see if input finally works.
 
----
+#### It doesn't work in a specific application
 
-### My cursor is stuck in osu!lazer (Wayland) {#osu-lazer-broken-input-wayland}
+Visit our wiki page for app-specific instructions [here]({% link _wiki/FAQ/LinuxAppSpecific.md %}).
 
-Make sure you set the `SDL_VIDEODRIVER` to `wayland`. Some examples:
+---
 
-```bash
-env SDL_VIDEODRIVER=wayland ./osu.AppImage
-```
+### My cursor is going crazy! It teleports everywhere!
 
-```bash
-env SDL_VIDEODRIVER=wayland osu-lazer
-```
+See [here]({% link _wiki/FAQ/General.md %}#emi-interference).
```

Comment:
we probably don't need this since it's in the general FAQ, maybe include a link to the generic FAQ at the bottom of the OS FAQ pages instead? I see it was added to the Windows FAQ too, if we're being methodical here we should also add it to the MacOS FAQ :^)